src/components/SurveyList/SurveyItem.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @next/next/no-img-element */
+import Link from 'next/link';
+import Image from 'next/image';
+
+import { Survey } from 'services/survey';
+
+import styles from './SurveyList.module.css';
+
+interface SurveyItemProps {
+  survey: Survey;
+}
+
+const SurveyItem = ({ survey }: SurveyItemProps) => {
+  return (
+    <>
+      <img
+        src={survey.coverImageUrl}
+        className="w-full h-auto max-h-[360px] mb-8 rounded-lg"
+        alt={`${survey.id} survey image`}
+      />
+      <div className="flex justify-between">
+        <div>
+          <h2 className="text-2xl mb-2">{survey.title}</h2>
+          <div className="text-gray-400">{survey.description}</div>
+        </div>
+        <Link href={`/surveys/${survey.id}`} passHref>
+          <div
+            className={styles.surveyItemLink}
+            data-testid={`survey-item-link-${survey.id}`}
+          >
+            <Image
+              width={20}
+              height={20}
+              src="/icon/chevron-right.svg"
+              alt="right chevron icon"
+            />
+          </div>
+        </Link>
```

Comment:
Should we make the entire survey card clickable? 🤔 
From what I see on most other websites, the whole product card, story, tweet, ... is clickable to ease navigation.

Reply:
Updated in [34a5526](https://github.com/hanam1ni/react-surveyor/pull/75/commits/34a5526aa6061380d83efb2e799b89da5a745cee)